packages/logs/src/domain/trackReports.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { isChromium } from '@datadog/browser-core'
+import { FAKE_REPORT, stubReportingObserver } from '../test/stubReportingObserver'
+import { trackReports } from './trackReports'
+
+describe('track reports', () => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏could we have a test for csp as well?

Reply:
I sure, I wanted first to be sure I was on track.

Reply:
Done